tidy3d/plugins/mode/solver.py

```diff
@@ -259,7 +259,7 @@ def conductivity_model_for_pec(eps, threshold=0.9 * pec_val):
         if np.any(eps_offd > TOL_TENSORIAL) or np.any(mu_offd > TOL_TENSORIAL):
```

Comment:
Setting a boolean flag from an if-clause looks like an anti-pattern:
```python
is_tensorial = np.any(eps_offd > TOL_TENSORIAL) or np.any(mu_offd > TOL_TENSORIAL)
```
Also below in `complex_solver`.

Reply:
Ah yeah thanks, who knows when this was a leftover from...